README.md

```diff
@@ -80,43 +82,55 @@ Empathy’s eDocs documentation portal.
   feature, Control your search history and have access to previous intentions.
   [![Release Docs](https://img.shields.io/badge/Released-July%202022-brightgreen)](https://empathy.co/blog/development-journey-my-history/)
-- Use [accessibility eslint plugin](https://github.com/vue-a11y/eslint-plugin-vuejs-accessibility)
+- Use
+  **[accessibility eslint plugin](https://github.com/vue-a11y/eslint-plugin-vuejs-accessibility)**
   to improve components a11y.
   [![Release Docs](https://img.shields.io/badge/Released-July%202022-brightgreen)](https://docs.empathy.co)
-- Next Queries Preview: A set of results that matches searches that other shoppers performed after
-  the current to be show within the SERP. This helps the shopper to discover interesting products
-  after the search action. This represents Empathy sciences for **Inspiration** and
+- **Next Queries Preview**: A set of results that matches searches that other shoppers performed
+  after the current to be show within the SERP. This helps the shopper to discover interesting
+  products after the search action. This represents Empathy sciences for **Inspiration** and
   **Cross-Selling**.
   [![Release Docs](https://img.shields.io/badge/Released-August%202022-brightgreen)](https://docs.empathy.co)
-- Query Results Preview: Provides a list of results matching a query to be inserted before the
-  shopper starts typing as inspiration, in the predictive layer for results matching the current
-  query, or as inspiration in a no results scenario.
+- **[Query Results Preview](https://docs.empathy.co/develop-empathy-platform/build-search-ui/web-archetype-integration-guide.html#setting-up-the-queries-preview-dynamically)**:
+  Provides a list of results matching a query to be inserted before the shopper starts typing as
+  inspiration, in the predictive layer for results matching the current query, or as inspiration in
+  a no results scenario.
   [![Release Docs](https://img.shields.io/badge/Released-September%202022-brightgreen)](https://docs.empathy.co/develop-empathy-platform/build-search-ui/web-archetype-integration-guide.html#setting-up-the-queries-preview-dynamically)
-- Design System Builder POC using Tailwind.
-  [![Release Docs](https://img.shields.io/badge/In%20Progress-Q3%202022-yellow)](https://github.com/empathyco/x/tree/main/packages/x-tailwindcss)
-- WCAG AA compliance.
-- Result Preview component to rapidly discover more result attributed without leaving the SERP.
-- Suggestions with filters that allows to select a query suggestion with a specific filter.
-- Provide a No Network State for components depending on API requests.
-- Incorporate search box power-ups such as animated suggestions or forbidden character set to
-  prevent code injection.
-- Project generator CLI.
+- **Search box power-ups** such as
+  [animated suggestions](https://docs.empathy.co/develop-empathy-platform/ui-reference/components/search-box/x-components.search-input-placeholder.html)
```

Comment:
This does not work on eDocs. Are they already aware?

Reply:
Uhmm..is working as far as i see, could it be something temporal?

Reply:
Yes, the placeholder live example is not working, but it's not relative to this PR. Thanks for the heads up.